src/showcase/showcase-changes.less

```diff
@@ -179,11 +179,15 @@
     }
 }
 
+.multi.rsi.indicator {
+    stroke: @tertiary-purple;
+}
```

Comment:
I'm somewhat surprised not to see a companion change for this? Were we just always using the wrong colour?